src/types/plugin-input.ts

```diff
@@ -1,23 +1,19 @@
 import { StaticDecode, Type as T } from "@sinclair/typebox";
-import { StandardValidator } from "typebox-validators";
-import { SupportedEvents, SupportedEventsU } from "./context";
-
-export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU extends SupportedEvents[T] = SupportedEvents[T]> {
-  stateId: string;
-  eventName: T;
-  eventPayload: TU["payload"];
-  settings: PluginSettings;
-  authToken: string;
-  ref: string;
-}
 
 export enum AssignedIssueScope {
   ORG = "org",
   REPO = "repo",
   NETWORK = "network",
 }
 
-const rolesWithReviewAuthority = T.Array(T.String(), { default: ["COLLABORATOR", "OWNER", "MEMBER", "ADMIN"] });
+export enum Role {
+  OWNER = "OWNER",
+  ADMIN = "ADMIN",
+  MEMBER = "MEMBER",
+  COLLABORATOR = "COLLABORATOR",
+}
+
+const rolesWithReviewAuthority = T.Array(T.Enum(Role), { default: [Role.OWNER, Role.ADMIN, Role.MEMBER, Role.COLLABORATOR], uniqueItems: true });
```

Comment:
With Typebox v33+, the array gets filled with default values even if it's already filled, for example the test for default supplies `[Role.OWNER, Role.ADMIN, Role.MEMBER]` and after `Value.Default` is called, the array is `[Role.OWNER, Role.ADMIN, Role.MEMBER, Role.COLLABORATOR]`.
This behaviour is not logical to me, but I can't find a way to make it behave like before other than downgrading to v32

Reply:
That would explain the issue I just had here: https://github.com/ubiquity-os-marketplace/text-conversation-rewards/pull/186

that's quite annoying indeed.

Reply:
@whilefoo Seems that it got taken care of! https://github.com/sinclairzx81/typebox/pull/1083

Reply:
That's awesome and we didn't even make an issue :D